src/app/[locale]/settings/_components/mnestix-connections/MnestixConnectionsCard.tsx

```diff
@@ -62,19 +71,41 @@ export function MnestixConnectionsCard() {
                     id: data.id,
                     url: data.url,
                     type: data.type.typeName,
+                    image: data.image,
+                    aasSearcher: data.aasSearcher,
+                    name: data.name
```

Comment:
IDE is throwing `TS2339: Property image does not exist on type` for image, aasSearcher and name, which I've read could lead to unexpected behavior (?)

Proposed changes:
```suggestion
                    image: data.image ?? undefined,
                    aasSearcher: data.aasSearcher ?? undefined,
                    name: data.name ?? undefined
```